common/src/main/java/whocraft/tardis_refined/client/screen/selections/ShellSelectionScreen.java

```diff
@@ -192,6 +192,11 @@ public GenericMonitorSelectionList createSelectionList() {
         for (Holder.Reference<ShellTheme> shellTheme : ShellTheme.SHELL_THEME_REGISTRY.holders().toList()) {
             ShellTheme theme = shellTheme.value();
             ResourceLocation shellThemeId = shellTheme.key().location();
+
+            if (theme == ShellTheme.HALF_BAKED.get()) {
+                continue;
+            }
```

Comment:
Why does this check need to be hardcoded?

Reply:
The default shell is no longer the factory, it is in a state that is "unfinished". We don't want that to appear in the monitor.